### PR TITLE
Wound refactor two compatability

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Makes https://github.com/Skyrat-SS13/Skyrat-tg/pull/23601 compatable

## How This Contributes To The Skyrat Roleplay Experience

Refactors are nice

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
No player facing changes
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
map modification????????

**Comment by nikothedude** [comment]
as before i reccomend this is test merged first

File: code/__DEFINES/wounds.dm [hunk 1]
```diff
@@ -195,16 +201,19 @@ GLOBAL_LIST_INIT(wound_series_collections, generate_wound_series_collection())
 /// Allows for determining of which wound series are caused by what.
 GLOBAL_LIST_INIT(wounding_types_to_series, list(
 	WOUND_BLUNT = list(
-		WOUND_SERIES_BONE_BLUNT_BASIC
+		WOUND_SERIES_BONE_BLUNT_BASIC,
+		WOUND_SERIES_MUSCLE_DAMAGE // SKYRAT EDIT -- MUSCLE WOUNDS
```
Comment (GoldenAlpharex) [inline code/__DEFINES/wounds.dm]: Always add trailing commas, it makes merge conflicts ever so slightly less likely to happen in the future.
```suggestion
		WOUND_SERIES_MUSCLE_DAMAGE, // SKYRAT EDIT -- MUSCLE WOUNDS
```

File: code/__DEFINES/wounds.dm [hunk 2]
```diff
@@ -195,16 +201,19 @@ GLOBAL_LIST_INIT(wound_series_collections, generate_wound_series_collection())
...
 	),
 	WOUND_SLASH = list(
 		WOUND_SERIES_FLESH_SLASH_BLEED,
+		WOUND_SERIES_MUSCLE_DAMAGE // SKYRAT EDIT -- MUSCLE WOUNDS
```
Comment (GoldenAlpharex) [inline code/__DEFINES/wounds.dm]:
```suggestion
		WOUND_SERIES_MUSCLE_DAMAGE, // SKYRAT EDIT -- MUSCLE WOUNDS
```

File: code/__DEFINES/wounds.dm [hunk 3]
```diff
@@ -195,16 +201,19 @@ GLOBAL_LIST_INIT(wound_series_collections, generate_wound_series_collection())
...
 	),
 	WOUND_BURN = list(
 		WOUND_SERIES_FLESH_BURN_BASIC,
 	),
 	WOUND_PUNCTURE = list(
-		WOUND_SERIES_FLESH_PUNCTURE_BLEED
+		WOUND_SERIES_FLESH_PUNCTURE_BLEED,
+		WOUND_SERIES_MUSCLE_DAMAGE // SKYRAT EDIT -- MUSCLE WOUNDS
```
Comment (GoldenAlpharex) [inline code/__DEFINES/wounds.dm]:
```suggestion
		WOUND_SERIES_MUSCLE_DAMAGE, // SKYRAT EDIT -- MUSCLE WOUNDS
```

File: modular_skyrat/modules/medical/code/wounds/muscle.dm [hunk 4]
```diff
@@ -74,14 +39,18 @@
...
 
-	update_inefficiencies()
+	return ..()
+
+/datum/wound/muscle/set_victim(new_victim)
+	if (victim)
+		UnregisterSignal(victim, COMSIG_HUMAN_EARLY_UNARMED_ATTACK)
+
+	if (new_victim)
+		RegisterSignal(new_victim, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, PROC_REF(attack_with_hurt_hand))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/medical/code/wounds/muscle.dm:49]:
```suggestion
		RegisterSignal(new_victim, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, PROC_REF(attack_with_hurt_hand))

```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**